core/src/main/java/io/questdb/cutlass/http/HttpResponseSink.java

```diff
@@ -539,9 +538,14 @@ public int getCode() {
             return code;
         }
 
+        public boolean isChunky() {
```

Comment:
nit: we already have `isChunked` method in `HttpClient`, so it'd be better to rename this one.

Reply:
renamed it to `isChunked()` for consistency